# Message used to send training related information
int32 episode_number
float32 episode_reward
